client/blocks/importer/wordpress/upgrade-plan/index.tsx

```diff
@@ -126,8 +141,42 @@ export const UpgradePlan: React.FunctionComponent< Props > = ( props: Props ) =>
 		);
 	};
 
+	const navigateBack = () => {
+		const queryParams = new URLSearchParams( window.location.search );
+		queryParams.delete( 'siteSlug' );
+		queryParams.delete( 'showModal' );
+		// Navigate back to site picker keeping necessary the query params.
+		window.location.assign( `sitePicker?${ queryParams.toString() }` );
+	};
+
 	return (
 		<div className="import__upgrade-plan">
+			{ showModal && (
+				<ConfirmModal
+					compact={ false }
+					title={ translate( 'Migration sounds daunting? It shouldn’t be!' ) }
+					confirmText="Take the deal"
```

Comment:
Lets add translation for `confirmText`.

Reply:
Maybe they will change since copies and design are not 100% final, but good to address just in case 👍